tests/compute/credentials.go

```diff
@@ -276,3 +216,21 @@ func withSSHPK(secretName string, propagationMethod v1.SSHPublicKeyAccessCredent
 		}
 	}
 }
+
+func withPassword(secretName string) libvmi.Option {
```

Comment:
We might have a `libvmi` Options for this and `withSSHPK` soon.

Reply:
https://github.com/kubevirt/kubevirt/pull/12791

It was merged. The `libvmi` options are called `WithAccessCredentialSSHPublicKey` and `WithAccessCredentialUserPassword`.